packages/@okta/vuepress-theme-prose/global-components/ErrorCodes.vue

```diff
@@ -41,275 +101,288 @@
-          select {
-            width: calc(100% - 25px);
-          }
+  @media screen and (max-width: 786px) {
+    .error-codes-search-container {
+      flex-direction: column;
+      input#error-code-search {
+        width: 100%;
+        margin: 0 0 10px 0;
+      }
+      .status-wrapper {
+        width: 100%;
+        select {
+          width: calc(100% - 25px);
         }
       }
     }
+  }
 
-    .reset-search {
-      cursor: pointer;
-    }
-
-    .reset-search::before {
-      content: '\f00d';
-      margin-left: 8px;
-      font-family: fontawesome;
-      text-align: center;
-    }
+  .reset-search {
+    cursor: pointer;
+  }
 
-    select {
-      height: 45px;
-      border: 2px solid #d2d2d6;
-    }
+  .reset-search::before {
+    content: "\f00d";
+    margin-left: 8px;
+    font-family: fontawesome;
+    text-align: center;
+  }
 
-    #error-code-search {
-      flex: 1;
-      margin-right: 10px;
-      border: 2px solid #d2d2d6;
-      font-size: 18px;
-      padding-top:10px;
-      padding-bottom: 10px;
-      padding-left: 10px;
-    }
+  select {
+    height: 45px;
+    border: 2px solid #d2d2d6;
+  }
 
-    #error-code-search::placeholder {
-      color: #d2d2d6;
-    }
+  #error-code-search {
+    flex: 1;
+    margin-right: 10px;
+    border: 2px solid #d2d2d6;
+    font-size: 18px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-left: 10px;
+  }
 
-    #error-code-release {
-      margin-top: 1em;
-    }
+  #error-code-search::placeholder {
+    color: #d2d2d6;
+  }
 
-    #error-code-count {
-      margin-top: -1em;
-      margin-left: 0.3em;
-      color: #888888;
-      font-size: 0.9em;
-    }
+  #error-code-release {
+    margin-top: 1em;
+  }
 
-    .error-code {
-      h4 {
-        margin: 25px 0 0;
-        padding: 6px 10px;
-        clear: left;
-        overflow: hidden;
-        border-left: 3px solid #007dc1;
-        color: #007dc1;
-        text-overflow: ellipsis;
-        white-space: nowrap;
+  #error-code-count {
+    margin-top: -1em;
+    margin-left: 0.3em;
+    color: #888888;
+    font-size: 0.9em;
+  }
 
-      }
+  .error-code {
+    h4 {
+      margin: 25px 0 0;
+      padding: 6px 10px;
+      clear: left;
+      overflow: hidden;
+      border-left: 3px solid #007dc1;
+      color: #007dc1;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
 
-      h4 .title-error-code {
-        font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
-      }
+    h4 .title-error-code {
+      font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+    }
 
-      h4::before {
-        content: '\f071';
-        margin-right: 8px;
-        font-family: fontawesome;
-      }
+    h4::before {
+      content: "\f071";
+      margin-right: 8px;
+      font-family: fontawesome;
+    }
 
-      pre {
-        padding: 5px 0px;
-        margin: 0px;
-        white-space: pre-line;
-      }
+    pre {
+      padding: 0.5rem 0.5rem 2rem 0.5rem;
```

Comment:
Do you know why we put larger padding on the bottom of the `pre` block here?

Reply:
good point, I think we can use only .5rem everywhere